### PR TITLE
CSI-1375 Solana integration

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by irshadCypherock** [comment]
Task linked: [CSI-1375 Cysync flow integration and server integration](https://app.clickup.com/t/37308523/CSI-1375)

**Review by Suraj-Tiwari — APPROVED** [review]

File: .npmrc [hunk 1]
```diff
@@ -1 +1 @@
-//registry.npmjs.org/:_authToken=${NPM_AUTH}
+//registry.npmjs.org/:_authToken=NPM_AUTH
```
Comment (Suraj-Tiwari) [inline .npmrc]: don't... leave it as it is

**Review by Suraj-Tiwari — CHANGES_REQUESTED** [review]

File: .npmrc (hunk 1 above)
Comment (ghost) [reply .npmrc]: In PowerShell, this won't work without the changes i.e. I will have to manually change the shell every time I want to commit or run any npm command in this directory

Comment (Suraj-Tiwari) [reply .npmrc]: set NPM_AUTH as empty string in env, in your desktop, with your changes ci will break completely

**Review by Suraj-Tiwari — DISMISSED** [review]

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**